src/symbol/mod.rs

```diff
@@ -1336,6 +1370,40 @@ mod tests {
             );
         }
 
+        #[test]
+        fn kind_1105() {
+            let data = &[
+                5, 17, 224, 95, 151, 0, 1, 0, 0, 100, 97, 118, 49, 100, 95, 119, 95, 97, 118, 103,
+                95, 115, 115, 115, 101, 51, 0, 0, 0, 0,
+            ];
+
+            let symbol = Symbol {
+                data,
+                index: SymbolIndex(0),
+            };
+            assert_eq!(symbol.raw_kind(), 0x1105);
+            assert_eq!(
+                symbol.parse().expect("parse"),
+                SymbolData::Label(LabelSymbol {
+                    offset: PdbInternalSectionOffset {
+                        offset: 0x975fe0,
```

Comment:
The linter is still complaining about this line, otherwise G2G.

```suggestion
                        offset: 0x0097_5fe0,
```